charts/tenant-base/chart/values.yaml

```
@@ -0,0 +1,72 @@
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

nginx won't work when running as non-root, let's switch to podinfo which at least can work :)
```suggestion
  repository: ghcr.io/stefanprodan/podinfo
  pullPolicy: IfNotPresent
  # Overrides the image tag whose default is the chart appVersion.
  tag: 6.2.1
```